src/Sentry/ExperimentalMetricsOptions.cs

```diff
@@ -0,0 +1,61 @@
+using Sentry.Internal;
+
+namespace Sentry;
+
+/// <summary>
+/// Settings for the experimental Metrics feature. This feature is preview only and will very likely change in the future
+/// without a major version bump... so use at your own risk.
+/// </summary>
+public class ExperimentalMetricsOptions
+{
+    /// <summary>
+    /// Determines whether code locations should be recorded for Metrics
+    /// </summary>
+    public bool EnableCodeLocations { get; set; } = true;
+
+    private IList<SubstringOrRegexPattern> _captureSystemDiagnosticsInstruments = new List<SubstringOrRegexPattern>();
+
+    /// <summary>
+    /// <para>
+    /// A list of Substrings or Regular Expressions. Any `System.Diagnostics.Metrics.Instrument` whose name
+    /// matches one of the items in this list will be collected and reported to Sentry.
+    /// </para>
+    /// <para>
+    /// These can be either custom Instruments that you have created or any of the built in metrics that are available.
+    /// </para>
+    /// <para>
+    /// See https://learn.microsoft.com/en-us/dotnet/core/diagnostics/built-in-metrics for more information.
+    /// </para>
+    /// </summary>
+    public IList<SubstringOrRegexPattern> CaptureSystemDiagnosticsInstruments
```

Comment:
In terms of defaults, we could capture none/some/all of [these build in metrics](https://learn.microsoft.com/en-us/dotnet/core/diagnostics/built-in-metrics). 

Are there quotas that we need to be mindful of (so are we potentially going to chew through someone's entire Sentry Quota if we start collecting [routing match attempt](https://learn.microsoft.com/en-us/dotnet/core/diagnostics/built-in-metrics-aspnetcore#metric-aspnetcoreroutingmatch_attempts) metrics?

If not, all of those built in metrics are potentially useful/interesting to people so we could enable these.

Reply:
I've updated this PR to include all of the built in metrics by default.

Reply:
Since this is experimental I think it's fine to include it by default but definitely worth keeping an eye on impact.
Cost is usually driven by cardinality so if the metrics have no tags it could be OK. cc @bitsandfoxes to own this conversation internally

Reply:
The metrics definitely have tags. For example, [`http.server.request.duration`](https://learn.microsoft.com/en-us/dotnet/core/diagnostics/built-in-metrics-aspnetcore#metric-httpserverrequestduration) has 7 attributes, which add dimensionality represented as tags in Sentry. In most situations, that will result in 1 different metric per route configured in the ASP.NET Core application... but that's just one example. 

Basically anything that's documented as an Attribute in the built in metrics documentation turns up as a tag in Sentry.